generate-stackbrew-pr.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -e
+. functions.sh
+
+GITHUB_USERNAME="nodejs-docker-bot"
+gitpath=../docker-images
+IMAGES_FILE="library/node"
+
+function updated() {
+	local versions
+	local images_changed
+
+	IFS=' ' read -ra versions <<< "$(IFS=','; get_versions)"
+	images_changed=$(git show --name-only "$TRAVIS_COMMIT" "${versions[@]}")
+
+	if [ -z "$images_changed" ]; then
+		return 1
+	else
+		return 0
+	fi
+}
+
+function message() {
+	echo "Node: $TRAVIS_COMMIT_MESSAGE"
+}
+
+function pr_payload {
+	echo '{
+		"title": "Node: "'"$TRAVIS_COMMIT_MESSAGE"',
+		"body": "Commit: nodejs/docker-node@"'"$TRAVIS_COMMIT"',
+		"head": "'"$GITHUB_USERNAME"':master",
+		"base": "master"
+	}'
+}
+
+if updated; then
+	# Set Git User Info
+	GITHUB_USER_INFO="$(curl -H "Authorization: token $GITHUB_API_TOKEN" https://api.github.com/user 2> /dev/null)"
+	GIT_AUTHOR_NAME=$(echo "$GITHUB_USER_INFO" | jq -r '.name')
+	GIT_AUTHOR_EMAIL=$(echo "$GITHUB_USER_INFO" | jq -r '.email')
+	GIT_COMMITTER_NAME=$GIT_AUTHOR_NAME
+	GIT_COMMITTER_EMAIL=$GIT_AUTHOR_EMAIL
+
+	export GIT_AUTHOR_NAME GIT_AUTHOR_EMAIL GIT_COMMITTER_NAME GIT_COMMITTER_EMAIL
```

Comment:
I wonder if the committer should be the one who merged the PR?

Reply:
Hmm i like the idea. I'll see of it can easily be done.

Reply:
Done!!!